docs/new-feature-tutorial.md

```diff
@@ -149,12 +149,17 @@ Since this feature also adds a checkbox to the admin page, and adds a
 new property the Realm model that can be modified from there, you also
 need to make changes to the `update_realm` function in the same file: :
 
-    # zerver/views/__init__.py
+    # zerver/views/realm.py
 
     def update_realm(request, user_profile,
       name=REQ(validator=check_string, default=None),
       restricted_to_domain=REQ(validator=check_bool, default=None),
-      invite_by_admins_only=REQ(validator=check_bool,default=None)):
+      invite_required=REQ(validator=check_bool, default=None),
+      invite_by_admins_only=REQ(validator=check_bool,default=None),
+      create_stream_by_admins_only=REQ(validator=check_bool, default=None),
+      allow_message_editing=REQ(validator=check_bool, default=None),
+      message_content_edit_limit_seconds=REQ(converter=to_non_negative_int, default=None),
+      default_language=REQ(validator=check_string, default=None)):
```

Comment:
Maybe to avoid having to keep updating this as we add more features, perhaps we should just have this at some point continue with `... (more arguments)` ?